### ISSUE TITLE
Bad validation results with unet

### ISSUE THREAD

**Issue by AsmaBenBrahem** [qubvel/segmentation_models#455]
Hello,
I have a dataset of 1000 images with respective binary masks and I wanted to try this library for training it with unet and pretrained imagenet powerful weights for semantic binary segmentation.  but the training take forever for me. it take one hour for only 4 epochs. Also the training loss is going down well but not the validation loss (I used jaccard loss), same for iou which reached the value 0.82 in the 4th epoch but the iuo_val is still stuck in the value 0.2.
Here is the code I used:
![image](https://user-images.githubusercontent.com/48836506/114325470-3bd8e780-9b28-11eb-8dde-f259baab4115.png)
![image](https://user-images.githubusercontent.com/48836506/114325510-5dd26a00-9b28-11eb-8a06-f65101e357dd.png)
and here is a picture of the training process:
![image](https://user-images.githubusercontent.com/48836506/114325555-9a9e6100-9b28-11eb-934a-b9c79db87668.png)
As you see an hour has passed and it is still in the 5th epoch and the validation results are very bad. Although I tried training with unet (with binary crossentropy loss) without this library  and without pretrained weights on the same dataset and got good results (iou=0.86 for test set) and took only 2 hours for 60 epochs. I don't know why it take forever using segmentation_models library.
Please If I messed up somewhere in the code or if you have any idea about what I can do to resolve this problem. Please tell me
